docs/integrations/amazon-aws/lambda.md

```diff
@@ -168,11 +168,14 @@ To configure a CloudTrail Source, perform these steps:
 
 ### Collect Amazon CloudWatch Metrics
 
-To collect Amazon CloudWatch Metrics, see [Amazon CloudWatch Source For Metrics.](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)
-
-AWS Namespace tag to filter in source for Lambda will be - **AWS/Lambda**
-
-* **Metadata: **Add an **account** field to the source and assign it a value which is a friendly name / alias to your AWS account from which you are collecting metrics. This name will appear in the Sumo Logic Explorer View. Metrics can be queried via the “account field”.
+* Sumo Logic supports collecting metrics using two source types:
```

Comment:
I see we have mentioned the same sentence twice, one is with the link and another one is without the link. Why do we have to write it twice?
Configure an AWS Kinesis Firehose for Metrics Source (Recommended)
       or
    * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)